### PR TITLE
Remove test for default GW policy data source

### PR DESCRIPTION
**Pull request by annakhm**
This test was only relevant for early NSX versions, and VPN configuration breaks the tests since one extra GW policy is added.
Signed-off-by: Anna Khmelnitsky <akhmelnitsky@vmware.com>

### CODE REVIEW

File: nsxt/policy_common.go [hunk 1]
```diff
@@ -452,7 +452,6 @@ func getPolicyRulesFromSchema(d *schema.ResourceData) []model.Rule {
 		}
 
 		ruleList = append(ruleList, elem)
-		seq = seq + 1
```
Comment (ksamoray) [inline nsxt/policy_common.go:455]: Is this related to the test removal? Or just piggybacking on the other change?

Comment (annakhm) [reply nsxt/policy_common.go:455]: no, unrelated, just some code cleaning

**Review by 2ez4szliu — APPROVED** [review]
